--- st.c	2025-08-09 20:00:58.350298234 +0700
+++ st.c	2026-02-18 05:00:17.093233472 +0700
@@ -1913,6 +1913,8 @@
 			if (narg > 1)
 				xsettitle(strescseq.args[1]);
 			return;
+    // see: https://github.com/yaaama/st-mrdotx/commit/0841790cb0773bdac2ba87cb8825350e5e09b4e1
+    case 8:
 		case 52: /* manipulate selection data */
 			if (narg > 2 && allowwindowops) {
 				dec = base64dec(strescseq.args[2]);
